staff/templates/staff_base.html

```diff
@@ -1,82 +1,211 @@
-{% load static %}
 <!DOCTYPE html>
-<html>
+{% load static %}
+{% load settings_tags %}
+<html lang="en">
 <head>
-  <title>Mission Bit</title>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="//cdn.muicss.com/mui-0.9.43/css/mui.min.css" rel="stylesheet" type="text/css" />
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="{% static  'css/mui_assets.css' %}"/>
+    <link rel="icon" href="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/mblogo.png?raw=true"> <!-- LOGO ON THE TAB BAR -->
+    <!-- MDL STUFF -->
+    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
+    <!-- -->
+    <script src="//cdn.muicss.com/mui-0.9.43/js/mui.min.js"></script>
+    <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 
-<style>
-.sidenav {
-  height: 100%;
-  width: 240px;
-  position: fixed;
-  z-index: 1;
-  top: 20px;
-  left: 0;
-  background-color: whitesmoke;
-  overflow-x: hidden;
-  padding-top: 100px;
-}
+    <title>Mission Bit</title>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/instafeed.js/1.4.1/instafeed.min.js"></script>
+     <script type="text/javascript">
+        var galleryFeed = new Instafeed({
+        get: "user",
+        userId: "{% settings_value "INSTAUSERID" %}",
+        accessToken: "{% settings_value "INSTATOKEN" %}",
+        resolution: "standard_resolution",
+        useHttp: "true",
+        limit: 5,
+        target: "instafeed",
+        template: '<h4 class="slot-description">\{\{location\}\}</h4>' +
+            '<a class="instafeedimg" href="\{\{image\}\}" target="_blank"><img src="\{\{image\}\}"></a>' +
+            '<h4 class="slot-description"></h4><h4 class="slot-description"><img src="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/heart-icon.png?raw=true" width="30" height="30" style="opacity: 0.75">\{\{likes\}\} ' +
+            '<img src="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/comment-icon.png?raw=true" style="opacity: 0.75" width="30" height="30">\{\{model.comments.count\}\}' +
+            '</h4><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.3" />'
+            // CHANGE TEMPLATE IF YOU WANT TO
+      });
+      galleryFeed.run();
+    </script>
+    <script type="text/javascript">
```

Comment:
We probably don't want to add jQuery for something this simple, most new projects use more modern libraries or no library at all.

If the script tag is last in the document you don't need to worry about `document.ready()` or equivalent as all of the DOM that the script references will already be there.

All of the jQuery functionality here is accessible pretty readily without a library and works in any browser that anyone would really conceivably use in 2019. Some pointers:

* [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector)
* [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)
* [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)